src/main/webapp/CityForm.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="UTF-8" />
+<title>Event Form</title>
+<link rel="stylesheet" href="style.css" />
+<div id="content">
+  <div id="location-container"></div>
+  <form action="/api/cities" method="post">
+    <h1>Enter a city</h1>
+    <p>ID:</p>
+    <textarea name="id" placeholder="1"></textarea>
```

Comment:
This shouldn't be needed if  the backend generates the ID.

Same comment on the other forms